### ISSUE TITLE
Use `ember deploy production` on Travis

### ISSUE THREAD

**Issue by roschaefer** [roschaefer/rundfunk-mitbestimmen#136]
I saw changes only after local deployment with `ember deploy production`. However both logs on my machine look quite similar, so I assume deployment has happened. Not sure if this is due to caching problems again.

**Comment by roschaefer** [comment]
probably caused by https://github.com/michaljach/ember-cli-deploy-scp/issues/11

**Comment by roschaefer** [comment]
Pretty sure, it's related to `ember-cli-deploy-scp`.

Temporarily solved by using `scp` on the [command line on Travis](michaljach/ember-cli-deploy-scp#11) - close and use `ember deploy production` as soon as michaljach/ember-cli-deploy-scp#11 is resolved

**Comment by michaljach** [comment]
It is fixed in current version 1.0.8. Cheers.

**Comment by roschaefer** [comment]
Resolved by #163, `ember deploy production` deploys new versions for real.
Yet, travis reports errors/deprecations regarding rsync:
```
Error: rsync exited with code 11
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): undefined
DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by roschaefer** [comment]
after updating `package.json` in the frontend, the deprecation warning took the gloves off

**Comment by roschaefer** [comment]
closed by ccbef2c86059f76f03f93a7210b09383f84fb572
